Fitter/interface/CategoryManager.h

```diff
@@ -132,9 +133,31 @@ void CategoryManager::mergeCategories(TString mrg_name, std::vector<TString> cat
         AnalysisCategory* cat = this->getCategory(s);
         to_merge.push_back(cat);
     }
+    
+    for (AnalysisCategory* cat_merge: to_merge) {
+        cat_merge->setAsimov();
+    }
+    
     AnalysisCategory* merged_cat = new AnalysisCategory(mrg_name,to_merge);
     merged_cat->setProcOrder(proc_order);
     this->all_cats[mrg_name.Data()] = merged_cat; // This might be a bit spicy...
+    this->cat_names.push_back(mrg_name);
+}
+
+
+void CategoryManager::renameCategory(TString new_name, TString old_name) {
+    if (this->hasCategory(new_name)) {
+        // Invalid new_name
+        std::cout << TString::Format("[WARNING] Unable to rename category. Category already exists: %s",new_name.Data()) << std::endl;
+        return;
+    }
+    if (!this->hasCategory(old_name)) {
+        std::cout << TString::Format("[WARNING] Unable to find category: %s",old_name.Data()) << std::endl;
+        return;
+    }
+    AnalysisCategory* cat = this->getCategory(old_name.Data());
+    cat->cat_name = new_name.Data();
+    this->all_cats[new_name.Data()] = cat;
 }
```

Comment:
Do you have an update on the status of using this class method? In my previous comment on this I was not very fond of adding this sort of method in its current implementation.

I skimmed through `overlay_operator_variations_wrapper.C` and couldn't find any instance where this method was being used, so do you still need it? Or did you find an alternative method that sidesteps the need for this method? If it's the latter, then I suggest we just remove this method altogether.